generation/2d_ldm/2d_ldm_tutorial.ipynb

```diff
@@ -0,0 +1,1526 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    max_epochs = 200
</pre><p>The same as above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>